### DISCUSSION TITLE
Vehicle setup management

### DISCUSSION THREAD

**Discussion by vahaldor** [Hyundai-Kia-Connect/kia_uvo#604] [Ideas]
I bought KIA SORENTO and started using this integration, working great! Thanks

Does the API allow you to control settings in the vehicle? Each turning on the car resets setting for speed limit warning, which is popping up all the time and is super annoying. Having automation to turn it off would help a lot of people.

**Comment by fuatakgun** [comment]
nope, we dont have a control for this.
